jscomp/super_errors/super_location.ml

```diff
@@ -117,6 +117,37 @@ let super_warning_printer loc ppf w =
   end
 ;;
 
+(* taken from https://github.com/BuckleScript/ocaml/blob/d4144647d1bf9bc7dc3aadc24c25a7efa3a67915/parsing/location.ml#L337 *) 
+(* the only modified part is in print_phanton_error_prefix *)
+let pp_ksprintf ?before k fmt =
+  let buf = Buffer.create 64 in
+  let ppf = Format.formatter_of_buffer buf in
+  Misc.Color.set_color_tag_handling ppf;
+  begin match before with
+    | None -> ()
+    | Some f -> f ppf
+  end;
+  kfprintf
+    (fun _ ->
+      pp_print_flush ppf ();
+      let msg = Buffer.contents buf in
+      k msg)
+    ppf fmt
+
+let print_phanton_error_prefix ppf =
+  (* modified from the original. We use only 2 indentations for error report
+    (see super_error_reporter above) *)
+  Format.pp_print_as ppf 2 ""
```

Comment:
Sorry didn't see the edit. The new one:

```
This is:
  foo
But somewhere else wanted:
  bar
```

Alright then, I'll patch the thing